guideline/cfh.md

```diff
@@ -23,6 +23,8 @@ help run PyCon Asia Pacific in 2017 in a location they are local to. PyCon APAC
 Asia-Pacific countries to come together to meet with fellow Python users. The
 same can be said for anyone adopting an exploratory perspective towards Python. 
 
+The above text proposes that a certain group 'owns' PyCon APAC. The below text also puts forth certain rules and requirements which would-be organizers needs to adhere to before they can be _considered_ as eligable to organize PyCon APAC. If we endorse a certain party, I think we also need to be clear on the responsibility that we hold as the party that endorses a certain organizer. We cannot just say "yes, you can hold PyCon APAC" but do not follow through on issues that might occur after that.
```

Comment:
Better break this to multiple lines.  It'll make commenting easier.

This paragraph doesn't look like part of the document, but your comment for the document, @iq8al ?